config/types.go

```diff
@@ -17,14 +17,26 @@ const (
 	DefaultRunMode RunMode = "default"
 	// DefaultLogMarkerHeaderName is the default log marker header name
 	DefaultLogMarkerHeaderName string = "X-CRS-Test"
+	// DefaultMaxMarkerRetries is the default amount of retries that will be attempted to find the log markers
+	DefaultMaxMarkerRetries int = 20
+	// DefaultMaxMarkerLogLines is the default lines we are going read back in a logfile to find the markers
+	DefaultMaxMarkerLogLines int = 500
 )
 
 // FTWConfiguration FTW global Configuration
 type FTWConfiguration struct {
-	LogFile             string          `koanf:"logfile"`
-	TestOverride        FTWTestOverride `koanf:"testoverride"`
-	LogMarkerHeaderName string          `koanf:"logmarkerheadername"`
-	RunMode             RunMode         `koanf:"mode"`
+	// Logfile is the path to the file that contains the WAF logs to check
```

Comment:
Can the path be relative? If so, relative to the working directory?

Reply:
Yes. Even in the examples listed in the README you have a relative path example.